queries.sql

```diff
@@ -7,4 +7,18 @@ SELECT date_of_birth FROM animals WHERE name IN ('Agumon', 'Pikachu');
 SELECT name, escape_attemps	FROM animals WHERE weight_kg > 10.5;
 SELECT * FROM animals	WHERE neutered IS true;
 SELECT * FROM animals	WHERE name NOT LIKE 'Gabumon';
-SELECT * FROM animals	WHERE weight_kg BETWEEN 10.4 AND 17.3
\ No newline at end of file
+SELECT * FROM animals	WHERE weight_kg BETWEEN 10.4 AND 17.3
+
+SELECT COUNT(*) FROM animals;
+SELECT COUNT(*) FROM animals WHERE escape_attempts = 0;
+SELECT AVG(weight_kg) FROM animals;
+SELECT name FROM animals WHERE escape_attempts = (SELECT MAX(escape_attempts) FROM animals);
```

Comment:
- [ ] Please make sure the `escape_attempts` is well written in your animals table.